### PR TITLE
Enable JSX syntax if the babel plugin is present

### PR DESCRIPTION
**Pull request by sunesimonsen**
This should be safe as babel parser supports JSX out of the box.
https://babeljs.io/docs/en/babel-parser

### CODE REVIEW

**Comment by alexjeffburke** [comment]
I’m happy with this but note that it may be a stopgap

**Comment by alexjeffburke** [comment]
Let me try again - I am happy with this. I think it’s a pragmatic thing to do but we may need to consider how to allow other options to the integrated parser as a more future facing thing.

**Comment by gustavnikolaj** [comment]
@alexjeffburke, sure - let's have a think about that. In the meantime I'll merge this :-)

**Comment by gustavnikolaj** [comment]
Released in 0.5.0
